cluster/juju/layers/kubernetes-worker/templates/ingress-replication-controller.yaml

```diff
@@ -1,4 +1,9 @@
 apiVersion: v1
+kind: ConfigMap
```

Comment:
Since we're calling out `${POD_NAMESPACE}` to where it's looking for the ConfigMap should we also mark this with a specific namespace? or are we going to rely on behavior of always going into the default namespace on this one?